net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -394,13 +409,14 @@ mwan3_set_general_iptables()
 					-p ipv6-icmp \
 					-m icmp6 --icmpv6-type 137 \
 					-j RETURN
-				# do not mangle outgoing echo request
-				$IPT6 -A mwan3_hook \
-					-m set --match-set mwan3_source_v6 src \
-					-p ipv6-icmp \
-					-m icmp6 --icmpv6-type 128 \
-					-j RETURN
-
+				if [ "$FIX_KERNEL_ACTUAL_SK_BUG" = "yes" ]; the
```

Comment:
Spelling error and use natural sh booleans:


```suggestion
				if "$FIX_KERNEL_ACTUAL_SK_BUG"; then
```